scripts/generator/templates/template-vaadin-spring-bom.xml

```diff
@@ -90,12 +90,12 @@
             <dependency>
                 <groupId>com.vaadin</groupId>
                 <artifactId>vaadin-lumo-theme</artifactId>
-                <version>${project.version}</version>
+                <version>${vaadin.material.theme.version}</version>
```

Comment:
This looks odd

Reply:
nice catch.. i will update it. Thanks

Reply:
Thanks, @Knoobie! I'll ask @ZheSun88 to step in...